### PR TITLE
Refactor into separate modules.

### PR DESCRIPTION
**Pull request by plathrop-sfdc**
In preparation for adding complexity in the form of Sentinel support and
a few Sentinel-related utility applications, break the existing code
into modules.

### CODE REVIEW

File: krux_redis/client.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                    log.debug('Redis: Found connected slave: %s', slave.name)
+                    return conn
+
+            # got here? No slaves are working, so use the master
+            log.debug('Redis: No slaves found - returning master')
+            stats.incr('redis.error.no_slave')
+            return self.get_master()
+
+
+class RedisInstance(object):
```
Comment (hanpeter) [inline krux_redis/client.py]: Any reason not to break this into `redis_instance.py`?

File: krux_redis/cli.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+from __future__ import absolute_import
+
+from krux.cli import get_group
+
+
+DEFAULT_TIMEOUT = 0.5   # in seconds
+
+
+# Designed to be called from krux.cli, or programs inheriting from it
+def add_redis_cli_arguments(parser):
```
Comment (hanpeter) [inline krux_redis/cli.py]: Current practice is to put this under `client.py` and build a shallow CLI under `cli.py`. Feel free to disagree since it is only there because it has been there before.

File: krux_redis/client.py (hunk 1 above)
Comment (plathrop-sfdc) [inline krux_redis/client.py]: Seemed tightly coupled to the other class in the file. Willing to be convinced otherwise?

File: krux_redis/cli.py (hunk 2 above)
Comment (plathrop-sfdc) [inline krux_redis/cli.py]: I'm going to say that's out of scope for my current work, which is focused mainly around getting the structure in place to do my Sentinel work.

Comment (hanpeter) [inline krux_redis/cli.py]: If you think that is out of scope, then that's fine. Can you just leave a `TODO`, then?

File: krux_redis/client.py (hunk 1 above)
Comment (hanpeter) [inline krux_redis/client.py]: My reason for breaking this into a separate file is purely readability and find-ability. I personally do not think this class is more tightly coupled then other classes that we have in other repos. YMMV

Comment (plathrop-sfdc) [inline krux_redis/client.py]: 6ca6abf

File: krux_redis/cli.py (hunk 2 above)
Comment (plathrop-sfdc) [inline krux_redis/cli.py]: d1200b8
